sailor/assetcentral/notification.py

```diff
@@ -82,12 +82,14 @@
     AssetcentralFieldTemplate('operator', 'operator', is_exposed=False),
 ]
 
+NOTIFICATION_FIELD_MAP = {ft.our_name: ft for ft in _field_templates}
+
 
 @_add_properties_ft
 class Notification(AssetcentralEntity):
     """AssetCentral Notification Object."""
 
-    _field_templates = _field_templates
+    _field_map = NOTIFICATION_FIELD_MAP
```

Comment:
Do we really need `NOTIFICATION_FIELD_MAP`?
Couldn't we just have `_field_map = {ft.our_name: ft for ft in _field_templates}` here, and use `Notification._field_map` everywhere we're now using `NOTIFICATION_FIELD_MAP`?

Reply:
It does not matter where the field_map is stored. I can change it.